angelsindustries/locale/en/logistic.cfg

```diff
@@ -33,8 +33,6 @@ angels-construction-roboport-vequip=Vehicle Construction Roboport
 angels-heavy-energy-shield-vequip=Vehicle Heavy Shield Generator
 angels-rocket-defense-equipment-vequip=Vehicle Rocket Defense
 
-[recipe-name]
-
 [recipe-category-name]
```

Comment:
Then this and ``[item-subgroup-name]`` can also go, right?

Reply:
We do leave them in empty, not that it hurts anyway..